tools/paas_dashboard/dashboards/paas-overview.erb

```diff
@@ -0,0 +1,19 @@
+<script type='text/javascript'>
+$(function() {
+  Dashing.numColumns = 3
+  Dashing.widget_base_dimensions = [600, 500]
```

Comment:
I still think, we should have a responsive implementation instead.

Different scenarios, will handle this differently, which may cause some grief.